rosdep/python.yaml

```
@@ -4780,6 +4780,10 @@ sphinxcontrib-bibtex-pip:
   ubuntu:
     pip:
       packages: [sphinxcontrib-bibtex]
+svgpathtools:
```

Per [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules):
```
If a package is only available via pip use the -pip extension.
```
```suggestion
svgpathtools-pip:
```

---

Ah, I discovered this is written [here](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules) but I read [this document](http://docs.ros.org/independent/api/rosdep/html/contributing_rules.html) instead for contributing.